chpl/chpl-service/src/main/java/gov/healthit/chpl/validation/listing/ListingValidatorFactoryImpl.java

```diff
@@ -1,70 +1,29 @@
 package gov.healthit.chpl.validation.listing;
 
 import org.apache.commons.lang3.StringUtils;
-import org.apache.logging.log4j.LogManager;
-import org.apache.logging.log4j.Logger;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
 import gov.healthit.chpl.domain.CertifiedProductSearchDetails;
-import gov.healthit.chpl.util.ChplProductNumberUtil;
 import gov.healthit.chpl.util.ErrorMessageUtil;
+import lombok.extern.log4j.Log4j2;
 
 @Service
+@Log4j2
 public class ListingValidatorFactoryImpl implements ListingValidatorFactory {
```

Comment:
Left comment in chat - can we remove ListingValidatorFactory and make this one "not an Impl"?